doc/en/components/inputs/mask-input.md

```diff
@@ -0,0 +1,145 @@
+---
+title: $Platform$ MaskInput | Infragistics
+_description: Infragistics' $Platform$ MaskInput allows the user to control input and format the visible value based on configurable mask rules
+_keywords: $Platform$ input, $ProductName$, Infragistics
+mentionedTypes: ['MaskInput']
+---
+
+## $Platform$ Mask Input Overview
+
+The $ProductName$ Mask Input is an input field that allows the developer to control user input and format the visible value, based on configurable rules. It provides different input options and ease in use and configuration.
+
+### $Platform$ Mask Input Example
+
+<code-view style="height: 150px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/inputs/mask-input-overview"
+           alt="$Platform$ Mask Input Overview Example"
+           github-src="inputs/mask-input/overview">
+</code-view>
+
+## Usage
+
+<!-- WebComponents -->
+First, you need to install the $ProductName$ by running the following command:
+
+```cmd
+npm install {PackageWebComponents}
+```
+<!-- end: WebComponents -->
+
+Before using the `IgcMaskInput`, you need to register it as follows:
+
+
+```ts
+import { defineComponents, IgcMaskInputComponent } from 'igniteui-webcomponents';
+
+defineComponents(IgcMaskInputComponent);
+```
+
+### Mask Rules
+The table bellow shows the supported built-in mask rules:
+
+| Mask Character | Description |
+| :--- | :--- |
+| 0 | requires a digit (0-9) |
+| 9 | requires a digit (0-9) or a space |
+| # | requires a digit (0-9), plus (+), or minus (-) sign |
+| L | requires a letter (a-Z) |
+| ? | requires a letter (a-Z) or a space |
+| A | requires an alphanumeric (0-9, a-Z) |
+| a | requires an alphanumeric (0-9, a-Z) or a space |
+| & | any keyboard character (excluding space) |
+| C | any keyboard character |
+
+### Applying Mask
+
+Applying the mask is pretty straightforward. All you need to do is provide a predetermined pattern to the `mask` property of the input.
+
+In the example below, we will apply a mask for a phone number with an extension code.
+
+```html
+<igc-mask-input id="mask-input" mask="(####) 00-00-00 Ext. 9999">
+    <igc-icon name="github" slot="prefix"></igc-icon>
+    <igc-icon name="github" slot="suffix"></igc-icon>
+    <span slot="helper-text">This is some helper text</span>
```

Comment:
Change helper text to something related to the sample, for example "Phone number" or something else.